components/i2c_output/oled128x64large/definition.json

```diff
@@ -0,0 +1,13 @@
+{
+  "displayName": "128x64 OLED (Large Font)",
+  "published": true,
+  "vendor": "Adafruit",
+  "productURL": "https://www.adafruit.com/product/4440",
+  "documentationURL": "https://learn.adafruit.com/monochrome-oled-breakouts",
+  "description": "Monochrome 128x64 I2C OLED Display with a larger font size.",
+  "i2cAddresses": [ "0x3C" ],
```

Comment:
PID url Is x32 version, Should be x64 like https://www.adafruit.com/product/938
I2C address should be  0x3D for 128x64, 0x3C for 128x32